common/changes/office-ui-fabric-react/feature-Calendar_Year_Selector_2018-10-17-04-37.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Addition of year picker to Calendar",
+      "type": "patch"
```

Comment:
Minor, there is at least 1 new prop to Calendar api surface

Reply:
Sorry - I need to read a bit of documentation clearly - what's the required action on this one? Cheers.

Reply:
Hi @gitfish , the type of change should be 'minor', since it adds properties to a public type.
'patch' is for fixes that do not change the public API.

Reply:
Just edit it with an editor. I will fix it for you.

Reply:
Thanks @cliffkoh